pyrealm/splash/solar.py

```diff
@@ -0,0 +1,190 @@
+"""The ``solar`` submodule provides functions and classes to calculate daily solar
```

Comment:
Maybe change the name of the submodule name ''solar'' from  double quotes to single quotes